networkx/algorithms/isomorphism/temporalisomorphvf2.py

```diff
@@ -0,0 +1,219 @@
+import networkx as nx
```

Comment:
Please put:  `from __future__ import absolute_import`.

The reason your 2.7 tests on TravisCI were working is because that line was missing. With it, all the builds fail.